### ISSUE TITLE
Add .Net source code to build csv_to_sql_loader.exe

### ISSUE THREAD

**Issue by aryoda** [martinkabe/RSQLS#5]
This R packages has a GPL license but is using a non-open binary-only exe file which is not only a security issue but also an unwanted "feature" in (free) open source software.

Even though it is your decision to make the source code for the exe file open source too I would appreciate this step...

This would also allow to fix errors and find weaknesses...

Thank!

**Comment by martinkabe** [comment]
Yes, I'll make source code of exe file open source too. I've been improving it a bit and that's why I haven't done it yet.

Thanks for your notes!

**Comment by aryoda** [comment]
I love to hear that. Bulk insert with high throughput is really an open issue in R, esp. if you are using mixed encodings in one column (e. g. UTF-8 and Latin-1). The `odbc` package has still an open issue (probably solved in the future).

**Comment by martinkabe** [comment]
Yes, I absolutely agree with you and that was the reason why I tried my own solution. I really love R and pushing data into SQL Server table was really huge bottleneck. Anybody else can use RSLQS package and I'll be very happy for any of suggestions how to improve it.

I've committed all .net classes I wrote so package is really open source now.

Thanks a lot for your comments, I really appreciate it!
